training/src/main/java/wolox/training/models/DTO/LibraryServiceDTO.java

```diff
@@ -0,0 +1,126 @@
+package wolox.training.models.DTO;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LibraryServiceDTO {
+
+    private String genre;
+    private String authors;
+    private String Image;
+    private String title;
+    private String subtitle;
+    private String publisher;
+    private String year;
+    private Integer pages;
+    private String isbn;
+
+    public void mapJsonToDto(
+        JsonNode genre,
+        JsonNode authors,
+        String Image,
+        String title,
+        String subtitle,
+        JsonNode publisher,
+        String year,
+        Integer pages,
+        String isbn) {
```

Comment:
Instead of having all this parameters, all coming from the same object, you should just pass the object and make the "decomposition" of the object in the method. 
Each time you have a method with too many parameters, stop and check if there's an option that allows you to encapsulate all those parameters in an object.